### PR TITLE
docs(server-communication): Resolve todo

### PR DESCRIPTION
**Pull request by adaojunior**

### CODE REVIEW

**Comment by chalin** [comment]
Thanks, LGTM

File: public/docs/_examples/server-communication/dart/pubspec.yaml [hunk 1]
```diff
@@ -11,7 +11,7 @@ dependencies:
   http: ^0.11.3+3
   jsonpadding: ^0.1.0
   stream_transformers: ^0.3.0+3
-  http_in_memory_web_api: ^0.0.1
+  http_in_memory_web_api: ^0.2.0
```
Comment (wardbell) [inline public/docs/_examples/server-communication/dart/pubspec.yaml]: What is this, @chalin ?  Otherwise this PR makes sense to me.

Comment (chalin) [inline public/docs/_examples/server-communication/dart/pubspec.yaml]: @wardbell : there was a bug in `http_in_memory_web_api` which the author fixed. This PR removes the temporary workaround and bumps up the library version to get the fixed version.

Comment (wardbell) [inline public/docs/_examples/server-communication/dart/pubspec.yaml]: what is `http_in_memory_web_api`? I guess this is all Dart so I shouldn't care ;-)

Comment (adaojunior) [inline public/docs/_examples/server-communication/dart/pubspec.yaml]: @wardbell this is a port of [a2-in-memory-web-api](https://github.com/wardbell/a2-in-memory-web-api) that you wrote.

Comment (chalin) [inline public/docs/_examples/server-communication/dart/pubspec.yaml]: Yes, it is an all-Dart counterpart.

**Comment by kwalrath** [comment]
I'll merge this now unless someone screams...

**Pull request merged**